src/Utilities/model.jl

```diff
@@ -934,3 +936,36 @@ macro model(model_name, ss, sst, vs, vst, sf, sft, vf, vft)
     end
     return code
 end
+
+const LessThanIndicatorSetOne{T} = MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, MOI.LessThan{T}}
+const LessThanIndicatorSetZero{T} = MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO, MOI.LessThan{T}}
+
+@model(Model,
+       (MOI.ZeroOne, MOI.Integer),
+       (MOI.EqualTo, MOI.GreaterThan, MOI.LessThan, MOI.Interval,
+        MOI.Semicontinuous, MOI.Semiinteger),
+       (MOI.Reals, MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives,
+        MOI.SecondOrderCone, MOI.RotatedSecondOrderCone,
+        MOI.GeometricMeanCone, MOI.ExponentialCone, MOI.DualExponentialCone,
+        MOI.PositiveSemidefiniteConeTriangle, MOI.PositiveSemidefiniteConeSquare,
+        MOI.RootDetConeTriangle, MOI.RootDetConeSquare, MOI.LogDetConeTriangle,
+        MOI.LogDetConeSquare),
+       (MOI.PowerCone, MOI.DualPowerCone, MOI.SOS1, MOI.SOS2,
+        LessThanIndicatorSetOne, LessThanIndicatorSetZero),
+       (),
+       (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction),
+       (MOI.VectorOfVariables,),
+       (MOI.VectorAffineFunction, MOI.VectorQuadraticFunction))
+
+@doc raw"""
```

Comment:
Nice one, I was wondering how we could document that struct that is creating inside a macro